app/scripts/modules/core/pipeline/config/stages/createLoadBalancer/createLoadBalancerExecutionDetails.html

```diff
@@ -0,0 +1,60 @@
+<div ng-controller="createLoadBalancerExecutionDetailsCtrl">
+    <execution-details-section-nav sections="configSections"></execution-details-section-nav>
+    <div class="step-section-details" ng-if="detailsSection === 'loadBalancerConfig'">
+        <div class="row">
+
+            <table class="table table-condensed">
+                <thead>
+                    <tr>
+                        <th>Account</th>
+                        <th>Name</th>
+                        <th>Region</th>
+                        <th ng-if="provider === 'aws'">Subnet</th>
```

Comment:
This should be based on whether any of the providers being deployed to support subnets - I believe GCP is working on them, not sure where Azure stands.

Take a look at https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/pipeline/config/stages/deploy/deployStage.js#L54 for an example of determining if that flag should be set.